image/image_histogram_44797041/image_histogram.py

```diff
@@ -0,0 +1,86 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.